nservicebus/scalability-and-ha/scale-out.md

```diff
@@ -45,9 +45,9 @@ In the first case the the scaling out happens by means of competing consumers. I
 
 ### MSMQ
 
-Because MSMQ does not allow performant remote receives in most cases scaling out requires sender-side round-robin distribution. When using MSMQ different instances are usually deployed to different (virtual) machines. The following routing file (see [file-based routing](/nservicebus/messaging/file-based-routing.md)) shows scaling out of the `Sales` endpoint. System administrators are able to spin-up new instances of the endpoint should the load increase and the only requirement is adding an entry to the routing file. No changes in the source code are required.
+Because MSMQ does not allow performant remote receives in most cases scaling out requires sender-side round-robin distribution. When using MSMQ different instances are usually deployed to different (virtual) machines. The following instance mapping file (see [scaling out with sender-side distribution](/nservicebus/msmq/scalability-and-ha/sender-side-distribution.md)) shows scaling out of the `Sales` endpoint. System administrators are able to spin-up new instances of the endpoint should the load increase and the only requirement is adding an entry to the instance mapping file. No changes in the source code are required.
```

Comment:
This is an (almost) direct copy of nservicebus/msmq/scalability-and-ha/index_overview_6.partial.md but this article is about general scalability/ha. Should this page give a short summary and then link to the MSMQ page rather than show any snippets?